internal/testutils/parser/grammar.ebnf

```diff
@@ -15,7 +15,7 @@ Variable       = Identifier | Letter "to" Letter .
 Interval       = Identifier "as" "[" Identifier "," Identifier "|" Identifier "]" .
 LinearTerm     = ( [ Digits ] Identifier ) | Digits .
 LinearExpr     = [ "-" ] LinearTerm { ( "+" | "-" ) LinearTerm } | "Σ" "(" Variables ")" .
-IntervalDemand = Identifier ":" Number .
+IntervalDemand = Identifier ":" Variable .
```

Comment:
Should actually be Identifier instead. Variables are allowed to be of the form "a to b" which we don't want here.